### PR TITLE
release note for v1.9

### PR DESCRIPTION
**Pull request by QuanluZhang**

### CODE REVIEW

File: docs/en_US/Release.md [hunk 1]
```diff
@@ -1,5 +1,49 @@
...
+
+* Support of configuring python environment using "preCommand" in remote mode (#2875)
+* Support AML training service in Windows (#2882)
+* Support reuse mode for remote training service (#2923)
+
+### WebUI & nnictl
+
+* Support the new design of the Overview page. (#2914)
+* Upgraded node, yarn and FabricUI, and enabled Eslint
+* Add/Remove columns in hyper-parameter chart and trial details table and Refactor trial details table (click event) (#2900)
```
Comment (ultmaster) [inline docs/en_US/Release.md:28]: Remove the refactor part.

Or at least write `refactor` instead of `Refactor`.

File: docs/en_US/Release.md [hunk 2]
```diff
@@ -1,5 +1,49 @@
...
+## Documentation
+* Refactor of the document for model compression (#2919)
+
+## Bug fixes
+* Bug fix of naïve evolution tuner, correctly deal with trial fails (#2695)
+* Resolve the warning "WARNING (nni.protocol) IPC pipeline not exists, maybe you are importing tuner/assessor from trial code?" (#2864)
+* Fix search space issue in experiment save/load (#2886)
+* Fix bug in experiment import data (#2878)
+* Fix annotation in remote mode (python 3.8 ast update issue) (#2881)
+
```
Comment (Lijiaoa) [inline docs/en_US/Release.md:46]: Could you help add a new item?
* Support customize trial's parameter is boolean when type is choice (#3003)

File: docs/en_US/Release.md [hunk 3]
```diff
@@ -1,5 +1,49 @@
...
+### Training service
+
+* Support of configuring python environment using "preCommand" in remote mode (#2875)
+* Support AML training service in Windows (#2882)
+* Support reuse mode for remote training service (#2923)
+
+### WebUI & nnictl
+
+* The Overview page on WebUI is redesigned with new layout (#2914)
```
Comment (Lijiaoa) [inline docs/en_US/Release.md]: (# 2924)

File: docs/en_US/Release.md (hunk 1 above)
Comment (QuanluZhang) [reply docs/en_US/Release.md]: removed

**Review by ultmaster — APPROVED** [review]

File: docs/en_US/Release.md [hunk 4]
```diff
@@ -1,5 +1,49 @@
...
+* Upgraded node, yarn and FabricUI, and enabled Eslint
```
Comment (Lijiaoa) [inline docs/en_US/Release.md:27]: node yarn(#2894)
FabricUI(#2873)
Eslint(#2744)

**Review by chicm-ms — APPROVED** [review]

File: docs/en_US/Release.md (hunk 2 above)
Comment (QuanluZhang) [reply docs/en_US/Release.md:46]: added

File: docs/en_US/Release.md (hunk 4 above)
Comment (QuanluZhang) [reply docs/en_US/Release.md]: added

**Review by Lijiaoa — APPROVED** [review]

**Pull request merged**